build-android/README.md

```diff
@@ -2,18 +2,48 @@
 
 Download the latest [build for Android](http://d.etpan.org/mailcore2-deps/mailcore2-android/)
 
-### Build for Android ###
+### Build on Mac ###
+Tested with macOS 10.13.6, Android Studio 3.1.4
 
+Install Java 6, 8, or 10
+* http://www.oracle.com/technetwork/java/javase/downloads/jdk10-downloads-4416644.html
```

Comment:
Could you try not being too specific with downloads URLs?
Java version will evolve over time and I'd like to avoid changing these instructions too often.